x-pack/plugins/security_solution/server/lib/detection_engine/rule_execution_log/event_log_adapter/constants.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const RULE_EXECUTION_LOG_PROVIDER = 'security-solution.rule-execution';
```

Comment:
The [RFC](https://docs.google.com/document/d/1mN5fhzaGgUimJB6lnNE5sGW7yyNTlot7asOn2B-S_iM/edit#heading=h.dk0cmk2u2q46) has the provider as `rule-execution.security`(?), and the lower section on providers doesn't seem to specify a specific format (though suggests a few). This is 👍 with me, but we should update the RFC with the specified format for other solutions to follow.

Reply:
It looks like this part of the RFC changed after I opened this PR 😅 Np, I'll adjust the string accordingly